### DISCUSSION TITLE
Provider tells me: bounce from SL direction happens because SL

### DISCUSSION THREAD

**Discussion by ledgeri** [simple-login/app#1333] [Q&A] [answered]
Hi!

I "statisticly contiinously" get more and more bounce-offs back to the SL bounce bucket -or at least feels like it-, and i dont know why. My huge bet is on my email-account-provider, because if the expected mail (some kind of activity notification, like "new stream-starts" from twitch) in it's structure does not change, it is out. I trust more on SL, so that is out too, but the mail provider changing owners, even if it is a "on paper only"-kind.
Also sometimes what was able to be fully delivered, it changes to bounced, what bounced normally, bounces broken, and vica-versa.

I asked them about it, they asked about the mails, where they do not arrive. Then i told the anomallies from above, and told them i recive many, but just some starts to do this (my calm avarage is 60 mails/two weeks, heavy can be 200+). Then they asked me to ask you about "THE DETAILS, because You are who can deliver it to them". No other details about the _details_, no names about what data-lines, or other stuff would be needed, just this basic line... what describes them more than anything...

So just for my lerning-sake asking: What data can help to them from you, what could fix that, if any? Or what can cause changing bounce behaviour, or chance, on any side? (If it is a ticket worthy conversation , i will do that, sure, but only if needed...)

Thanks for the reply, and for the service!

**Comment by nguyenkims** [comment, marked as answer]
@ledgeri can you please contact our support at https://app.simplelogin.io/dashboard/support and give us info like the alias, the timestamp when that happened so we can investigate?

**Reply by ledgeri** [reply]
Sure thing, but please, for as a searchable public answer: Could you give a broad answer? 
Or it is must be a a special case... so out of any possible category...

(Ticket activated)

**Reply by nguyenkims** [reply]
Unfortunately I don't see a known pattern in what you described so this will be rather a case by case investigation.

**Comment by ledgeri** [comment]
Just to update: 
SL support concluded: 20% of the mails forwarded to my provider are bouncing, because a policy issue on the provider's side.
Provider-questoning is pending right now.
